build-and-run.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 
+set -euf -o pipefail
+
```

Comment:
While I partly agree with having this here, this should be in a separate PR.
```suggestion
```

Reply:
Also, the `-u` flag causes issues if we don't specify all the variables, which is something that the script should handle fine.

Reply:
Yeah I didn't realize I staged build.sh and build-and-run.sh, changed them back to what they were previously.